CONTRIBUTING.md

```diff
@@ -55,3 +55,7 @@ Make sure to run the complete suite before submitting a PR
 ```
 pytest
 ```
+
+## Making a pull request
+
+We try to follow [Conventional Commit](https://www.conventionalcommits.org/) for commit messages and PR titles. Since we merge PR's using squash commits, it's fine if the final commit message (in the PR body) follows this convention.
```

Comment:
While this is true, we don't want to suggest that only one commit message will be part of the PR commit log though, as we should encourage descriptive and helpful messages in the history. Maybe
```suggestion
We try to follow [Conventional Commit](https://www.conventionalcommits.org/) for commit messages and PR titles. Since we merge PR's using squash commits, it's fine if the final commit messages (proposed in the PR body) follow this convention.
```
?